sdk/database/dbplugin/v5/plugin_server.go

```diff
@@ -31,7 +31,36 @@ func ServeConfig(db Database) *plugin.ServeConfig {
 	}
 
 	conf := &plugin.ServeConfig{
-		HandshakeConfig:  handshakeConfig,
+		HandshakeConfig:  HandshakeConfig,
+		VersionedPlugins: pluginSets,
+		GRPCServer:       plugin.DefaultGRPCServer,
+	}
+
+	return conf
+}
+
+func ServeMultiplex(factory func() (Database, error)) {
```

Comment:
I am wondering if this file can be merged into main in a separate PR since the Oracle database plugin will need the updated SDK?